docs/source-app/workflows/byoc/index.rst

```diff
@@ -0,0 +1,115 @@
+
+#################################
+Run Apps on your own cloud (BYOC)
+#################################
+
+**Audience:** Users looking to run Lightning Apps on their own private cloud.
+
+----
+
+*******************
+A bit of background
+*******************
+
+BYOC - Bring Your Own Cloud, is an alternate deployment model to Lightning Cloud (fully managed SaaS).
```

Comment:
nit: I think we use `BYOC` as "Bring your own credentials"

Reply:
Will verify terms